src/components/FlowPlayer/FlowPlayer.tsx

```diff
@@ -38,6 +39,7 @@ export const FlowPlayer: FunctionComponent<FlowPlayerProps> = ({
 	token,
 	dataPlayerId,
 	seekTime,
+	autoplay = true,
```

Comment:
by default autoplay? Ik zou de default op false zetten
als we bv meerdere videos op een pagina hebben

Reply:
@bertyhell In principe zou dit geen probleem mogen zijn, want hij kan maar gestart worden na het triggeren van de onInit bij het klikken van de video.

Enkel in de FlowPlayer content block zal dit op false moeten lijkt me.